packages/story-utils/src/generateSnapshots.ts

```diff
@@ -1,53 +1,57 @@
+import type { Meta, Story, StoryContext } from '@storybook/react';
+import { composeStories } from '@storybook/testing-react';
 import { render, RenderResult } from '@testing-library/react';
-import { getStoriesFromStoryFileExports } from './getStories';
-import type { StoryData, StoryFileExports } from './getStories';
-import prepareStory from './prepareStory';
+import { createElement } from 'react';
 import wait from './wait';
 
-type TestOptions<Args> = {
+type TestOptions = {
   /**
-   * get the snapshot either synchronously, returning the ChildNode,
-   * or asynchronously returning a promise that resolves to the ChildNode.
+   * Get the element to snapshot, either synchronously or asynchronously.
    */
-  getSnapshot?: (
+  getElement?: (
```

Comment:
Renamed from getSnapshot to getElement, because you use it to get the element you want to snapshot.